src/previous.js

```diff
@@ -0,0 +1,40 @@
+import { parse, stringify } from 'qs';
```

Comment:
Can we use the standard [URL.searchParams](https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams) method here rather than adding another dependency? Our [IE 11 polyfill](https://polyfill.io/v2/docs/features/) should support the URL API.